synapse/lib/certdir.py

```diff
@@ -100,6 +101,15 @@ def _saveCertTo(self, cert, *paths):
             fd.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert))
         return path
 
+    def _saveClientCertTo(self, cert, *paths):
```

Comment:
This would look something like:

```
    def _saveCertTo(self, cert, *paths):
        path = self.getPathJoin(*paths)
        if os.path.isfile(path):
            raise s_common.DupFileName(path=path)

        data = None
        if isinstance(cert, type1):
             data = '123'
        elif isinstance(cert, type2):
             data = '456'

        if data:
            with s_common.genfile(path) as fd:
                fd.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert))
        return path
```